tutorials/machine_learning/index.md

```diff
@@ -0,0 +1,140 @@
+\addtogroup tutorial_ml
+
+
+## Table of contents
+- [Basic TMVA tutorials](\ref basic)
+   - [Training](\ref training)
+   - [Applications](\ref application)
+   - [Others](\ref other)
+- [Cross validation](\ref cross_val)
+- [New TMVA interfaces](\ref new_interface)
+   - [RBDT](\ref rbdt)
+- [Deep learning in TMVA](\ref deep_learing)
+- [TMVA Keras tutorials](\ref keras)
+- [TMVA PyTorch tutorials](\ref pytorch)
+- [Interference with SOFIE](\ref interference)         
+- [RBatchGenerator](\ref rbatchgen)
```

Comment:
Is this our final choice for this name @vepadulano ?

Reply:
Data loading for training would be better :+1: